docs/src/performance.md

```diff
@@ -167,3 +168,89 @@ As a rule of thumb:
 - Do not use `@nospecialize` in performance-critical parts, in particular not for methods involved
   in computing `Trixi.rhs!`.
 - Consider using `@nospecialize` for methods like custom implementations of `Base.show`.
+
+
+## Performance metrics of the `AnalysisCallback`
+The [`AnalysisCallback`](@ref) computes several values that you can use to
+evaluate the serial and parallel performance of Trixi. Since they represent
+measured run times that are normalized by the number of `rhs!` evaluations and
+the number of degrees of freedom of the problem setup, we refer to them
+as "performance indices" or "PIDs". The normalization ensures that we can
+compare different measurements for each type of PID independent of the number of
+time steps or mesh size. All PIDs have in common that they are still in units of
+time, thus *lower is better* for each of them.
+
+Here, the term "degrees of freedom" (DOFs) refers to the number of *independent*
+state vectors that are used to represent the numerical solution. For example, if
+you use a DGSEM-type scheme in 2D on a mesh with 8 elements and with
+5-by-5 Gauss-Lobatto nodes in each element (i.e., a polynomial degree of 4), the
+total number of DOFs would be
+```math
+n_\text{DOFs,DGSEM} = \{number of elements\} \cdot \{number of nodes per element\} = 8 \cdot 5 \cdot 5 = 200.
+```
+Similarly, for a finite volume-type schemewith 64 elements, the total number of
+DOFs would be (independent of the number of spatial dimensions)
+```math
+n_\text{DOFs,FV} = \{number of elements\} = 64,
```

Comment:
```suggestion
n_\text{DOFs,FV} = \{number of elements\} = 8,
```
You use 8 elements above?

Reply:
Yes, but I changed it to 64 later. Using 8 consistently makes more sense didactically, so I changed it in 349b365f.